jcl/source/windows/JclDebug.pas

```diff
@@ -3498,6 +3545,73 @@ function TJclBinDebugScanner.SourceNameFromAddr(Addr: DWORD): string;
     Result := '';
 end;
 
+function TJclBinDebugScanner.VAFromUnitAndProcName( UnitName, ProcName : string ): DWORD;
+var
+  P          : Pointer;
+  VA         : Cardinal;
+  i,
+  Value,
+  FirstWord,
+  SecondWord : Integer;
+  S          : string;
+begin
+  Result := 0;
+  if ( UnitName = '' ) or ( ProcName = '' ) then
+    Exit;
+  ProcName := UnitName + '.' + ProcName;
+
+  if FCacheData then
+    begin
```

Comment:
Please respect indentation rules, `begin` is flush with the parent statement (`if` in that case)